DFC.Integration.AVFeed/DFC.Integration.Azure.Resources/App Service Plan/parameters.json

```diff
@@ -1,30 +0,0 @@
-{
```

Comment:
These files under DFC.Integration.Azure.Resources should not be deleted, they are required for deployment - Infrastructure as code.